notebooks/compute_ancestor_mapping.ipynb

```diff
@@ -3,7 +3,11 @@
   {
    "cell_type": "markdown",
    "id": "89ea87d1-2459-4c09-b482-786db8cd3031",
-   "metadata": {},
+   "metadata": {
+    "pycharm": {
+     "name": "#%% md\n"
```

Comment:
GH pipeline was throwing a linting error until I removed an extra newline at the end...

Reply:
Feel free to ignore when reviewing, no real diff